src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -362,15 +362,20 @@ def container_export(cmd, resource_group_name, name, file):
     resource.pop('kind', None)
     resource.pop('managed_by', None)
     resource['properties'].pop('provisioningState', None)
+    if 'ipAddress' in  resource['properties']:
+        resource['properties']['ipAddress'].pop('ip', None)
 
     for i in range(len(resource['properties']['containers'])):
         resource['properties']['containers'][i]['properties'].pop('instanceView', None)
 
     # Add the api version
     resource['apiVersion'] = container_group_client.api_version
 
-    with open(file, 'w+') as f:
-        yaml.dump(resource, f, default_flow_style=False)
+    if file:
+        with open(file, 'w+') as f:
+            yaml.dump(resource, f, default_flow_style=False)
+    else:
+        yaml.dump(resource, sys.stdout, default_flow_style=False)
```

Comment:
I don't know about this. This essentially bypasses out output pipeline. Note that all the commands' outputs go through a formatting process, therefore, the output is in user-specified format (`json`, `table`, and `tsv`). 

Is this a critical scenario in which we have to output this to the stdout as is?

Reply:
This has been a request from users since json is an extremely hard format to read. This allows users to see their container groups quickly vs having to go through the json.

Reply:
In that case, you can write a customized table formatter and suggest users to use `-otable` format.

Reply:
Overall, JSON is a better format for machine consumption, while table is a better format for human consumption. It is worth to optimize your commands' table formatting output if the output is intended to be heavily read by eyes.

Reply:
Chiming in as a heavy azure-cli user. Please don't break output formatting on individual commands. Using `-o table` or `--query '[].name'` would be much preferred to having to somehow know that a particular command didn't conform to the azure-cli conventions

Reply:
@noelbundick thank you for the feedback.